src/cmdlib.sh

```diff
@@ -93,12 +93,16 @@ depcheck() {
 preflight() {
     depcheck
 
-    if [ "$(stat -f --printf="%T" .)" = "overlayfs" ]; then
-        fatal "$(pwd) must be a volume"
-    fi
+    if [ -n "${BUILD:-}" ]; then
+        info "COSA is running as a buildconfig, skipping checks"
```

Comment:
Hmm.  I think I'd say we should make these into explicit options via environment variable instead; something like `COSA_SKIP_SANITY=kvm,volume` ?

Running without `/dev/kvm` probably works (qemu full emulation) but it's going to be ~10x slower and particularly painful for tests.  I think that's only something we'd want people to explicitly opt in to.

Reply:
I used the `BUILD` envVar since its specified in `buildconfig.openshift.io/v1` and gets set to via the OCP build operator. In fact, is the JSON buildConfig object. 

However, I see a way to do this in a saner way. Once `entrypoint` establishes that its on an OpenShift Build, *then* have it set the envVars. Fix up to follow.

Reply:
I split out https://github.com/coreos/coreos-assembler/pull/1795